app/helpers/repository_datatable_helper.rb

```diff
@@ -71,29 +72,29 @@ def default_table_columns
   end
 
   def display_cell_value(cell, team)
-    value_type = cell.value_type.underscore
-    if (DisplayCellValue.respond_to? value_type) && defined?(render)
-      render DisplayCellValue.public_send(value_type, cell, team)
-    elsif DisplayCellValue.respond_to? "#{value_type}_text"
-      DisplayCellValue.public_send("#{value_type}_text", cell, team)
-    else
-      custom_auto_link(display_tooltip(cell.value.data, Constants::NAME_MAX_LENGTH),
-                       simple_format: true,
-                       team: team)
-    end
+    cell_type = cell.value_type.underscore
```

Comment:
We can create separate serializers and call them by cell.value_type. Each data type should have implemented serializers, so there is no need to check if method present

Reply:
I created serializes for exist  column types